types/types.go

```diff
@@ -324,6 +324,8 @@ type SystemContext struct {
 	DockerDisableV1Ping bool
 	// Directory to use for OSTree temporary files
 	OSTreeTmpDirPath string
+	// Path to the system-wide registries configuration file
+	SystemRegistriesConfPath string
```

Comment:
Please move the field above the os-tree related one (which should have been in its own `=== ostree.Transport overrides ==` section).

(Assuming that `sysregistries` is Docker-specific; if not, to the general section.)

Reply:
Moved to general ... it is not docker specific.